### PR TITLE
Viite 1954 library updates with devtest integration

### PR DESCRIPTION
**Pull request by Campanha**

### CODE REVIEW

**Comment by Campanha** [comment]
UI tests are at least starting to run even tought 3 are failing out of 48

**Comment by Campanha** [comment]
In Jenkins "Run tests" Job will now run also UI tests. So if some of them fail it will complain and fail like in build http://livibuild04.vally.local/jenkins/job/viite/job/Run-Tests/99/console

**Comment by Campanha** [comment]
after changes:
Fixture-reset OK: http://livibuild04.vally.local/jenkins/job/viite/job/Fixture-Reset/147/console
Tests OK: http://livibuild04.vally.local/jenkins/job/viite/job/Run-Tests/102/console

**Comment by skosonen** [comment]
All the library updates were reverted from master and they are not included in this pull request either. Those library updates should be added here too and merged back to master together with these changes.

**Comment by andre-policarpo** [comment]
Git master over here :)
@Ricardo you've to double-check everything please.

**Comment by Campanha** [comment]
OK tests passing: http://livibuild04.vally.local/jenkins/job/viite/job/Run-Tests/103/console

File: src/main/scala/fi/liikennevirasto/digiroad2/DigiroadServer.scala [hunk 1]
```diff
@@ -127,12 +127,11 @@ class VKMProxyServlet extends ProxyServlet {
...
   def regex: Regex = "/(digiroad|viite)".r
 
-  override def rewriteURI(req: HttpServletRequest): java.net.URI = {
-
+  override def rewriteTarget(req: HttpServletRequest): String = {
     val properties = new Properties()
     properties.load(getClass.getResourceAsStream("/digiroad2.properties"))
     val vkmUrl: String = properties.getProperty("digiroad2.VKMUrl")
-    java.net.URI.create(vkmUrl + regex.replaceFirstIn(req.getRequestURI, ""))
+    vkmUrl
```
Comment (skosonen) [inline src/main/scala/fi/liikennevirasto/digiroad2/DigiroadServer.scala]: Is this VKM URL correct after these changes?

Comment (skosonen) [reply src/main/scala/fi/liikennevirasto/digiroad2/DigiroadServer.scala]: We can test this in devtest. It is easy to fix if needed.

**Pull request merged**